src/handlers/actions-handler.ts

```diff
@@ -21,6 +21,11 @@ export default class ActionsHandler {
   }
 
   COMMAND_TYPE_CLICK(data: any): Promise<any> {
+    if (data.text) {
+      return this.resolvePostMessage!("domClick", {
+        query: data.text,
+      });
+    }
```

Comment:
I think this is the same command overloading issue—it's a bit confusing that the behavior changes based on whether `text` or `path` is set, so it might be clearer to just have dedicated command types for clicking DOM elements.

Reply:
Agreed. This also avoids the confusion in focus commands that could come from this overloading. I added new command types to the client and added new handlers in e164113